src/Microsoft.DotNet.ProjectJsonMigration/transforms/IncludeContextTransform.cs

```diff
@@ -21,7 +21,9 @@ public class IncludeContextTransform : ConditionalTransform<IncludeContext, IEnu
                     itemName,
                     includeContext => FormatGlobPatternsForMsbuild(includeContext.IncludeFiles, includeContext.SourceBasePath),
                     includeContext => FormatGlobPatternsForMsbuild(includeContext.ExcludeFiles, includeContext.SourceBasePath),
-                    includeContext => includeContext != null && includeContext.IncludeFiles.Count > 0);
+                    includeContext => includeContext != null 
```

Comment:
I could and it would be more compact although I don't think more readable. 
IMO it's not a compelling enough of a change to block the PR on.